_includes/contact_form.html

```diff
@@ -2,152 +2,226 @@
   <div class="lg-col lg-col-12 select">
-    <label class='bold' for="00Nt0000000OYA2">{% t contact_page.support_request_form.labels.agency_application %}</label>
-    <select class='block sm-col-6 mb2 field select required' id="00Nt0000000OYA2" name="00Nt0000000OYA2" title="Agency Application" required="true" placeholder="Select Agency">
-      <option value="" selected hidden>{% t contact_page.support_request_form.form_helpers.select_one %}</option>
-      <option value="USAJobs">{% t contact_page.support_request_form.agencies.usajobs %}</option>
-      <option value="Trusted Traveler Program">{% t contact_page.support_request_form.agencies.trusted_traveler_program %}</option>
-      <option value="SAM.gov">{% t contact_page.support_request_form.agencies.sam %}</option>
+    <label class='bold' for="00Nt0000000OYA2">{% t contact_page.support_request_form.labels.agency_application
+      %}</label>
+    <select class='block sm-col-6 mb2 field select required' id="00Nt0000000OYA2" name="00Nt0000000OYA2"
+            title="Agency Application">
+      <option value="" selected hidden>{% t contact_page.support_request_form.form_helpers.select_one %}
+      </option>
+      <option value="USAJOBS">{% t contact_page.support_request_form.agencies.usajobs %}</option>
+      <option value="Trusted Traveler Programs (Global Entry/Nexus/Sentri)">{% t
+        contact_page.support_request_form.agencies.trusted_traveler_program %}
+      </option>
+      <option value="SAM">{% t contact_page.support_request_form.agencies.sam %}</option>
       <option value="CBP Jobs">{% t contact_page.support_request_form.agencies.cbp_jobs %}</option>
-      <option value="MyCBP Employee Portal">{% t contact_page.support_request_form.agencies.my_cbp_employee_portal %}</option>
+      <option value="National Registry of Certified Medical Examiners">{% t
+        contact_page.support_request_form.agencies.dot_nrcme %}
+      </option>
       <option value="CBP ROAM">{% t contact_page.support_request_form.agencies.cbp_roam %}</option>
-      <option value="DoT NRCME">{% t contact_page.support_request_form.agencies.dot_nrcme %}</option>
-      <option value="DoT Portal">{% t contact_page.support_request_form.agencies.dot_portal %}</option>
-      <option value="Move.mil">{% t contact_page.support_request_form.agencies.move_mil %}</option>
-      <option value="NGA">{% t contact_page.support_request_form.agencies.nga %}</option>
-      <option value="Railroad Retirement Board">{% t contact_page.support_request_form.agencies.railroad_retirement_board %}</option>
-      <option value="Other">{% t contact_page.support_request_form.agencies.other %}</option>
+      <option value="HOMES.mil">{% t contact_page.support_request_form.agencies.homes_mil %}</option>
+      <option value="Railroad Retirement Board">{% t
+        contact_page.support_request_form.agencies.railroad_retirement_board %}
+      </option>
+      <option value="Other">Other</option>
```

Comment:
Looks like we dropped the translation for "Other" here

Reply:
Fixed